app/uk/gov/hmrc/gform/formtemplate/DestinationsValidator.scala

```diff
@@ -51,4 +53,31 @@ object DestinationsValidator {
 
   def validate(template: FormTemplate): Opt[Unit] =
     validateUniqueDestinationIds(template.destinations).toEither
+
+  def nonExistentFieldId(nonExistentFieldIds: List[FormComponentId]) =
+    s"Pdf FieldId(s) ${nonExistentFieldIds.mkString(", ")} doesn't exist."
+
+  def validatePdfFieldIds(formTemplate: FormTemplate): Opt[Unit] = {
+    val allFormComponentIds: List[FormComponentId] = fieldIds(formTemplate.sections)
+
+    val pdfFieldIds: Option[List[FormComponentId]] = formTemplate.destinations match {
```

Comment:
`pdfFieldIds` should be just `List[FormComponentId]`.

Type `Option[List[T]]` is a code smell. Side note: If list can't be empty but is still optional than we should at least represent that as `Option[NonEmptyList[T]]`.